_data/navigation.yml

```
@@ -208,8 +208,8 @@ versions:
       #   url: /docs/releases/release-3.5/
       # - title: "v3.4"
       #   url: /docs/releases/release-3.4/
-      # - title: "Release Support Policy"
-      #   url: /docs/releases/release-support-policy
+      - title: "Release Support Policy"
+        url: /docs/releases/release-support-policy
```

```suggestion
        url: /docs/releases/release-support-policy/
```
This is a very minor thing, but is it better to add a trailing slash here, to keep consistency in this document?

---

Good catch! I've added a trailing slash for consistency.